yarn.lock

```diff
@@ -1666,6 +1666,11 @@
   resolved "https://registry.yarnpkg.com/@endo/eventual-send/-/eventual-send-0.16.5.tgz#39f2e21b5331cd9e423a8440f9e74e28fea6701e"
   integrity sha512-tWmia2jv+beGf4oVOxUqcHc358PX3Gssz4qUD3BA7D6d7AW75Pk34ArWsi29E9w6aNMrnhnpX+EaRBWoj/Q1PA==
 
+"@endo/eventual-send@^0.16.6":
+  version "0.16.6"
```

Comment:
Duplicate versions of eventual send is probably not right. Please dedup or revert to 0.16.5

Reply:
Fixed thanks. I blindly took the latest version not realizing the sdk was on 0.16.5